openvpn/check.go

```diff
@@ -18,31 +18,50 @@
 package openvpn
 
 import (
+	"bufio"
+	"bytes"
 	"errors"
+	log "github.com/cihub/seelog"
+	"io"
+	"net/textproto"
 	"os/exec"
 	"strconv"
 	"syscall"
 )
 
 // CheckOpenvpnBinary function checks that openvpn is available, given path to openvpn binary
 func CheckOpenvpnBinary(openvpnBinary string) error {
-
-	process := NewProcess(openvpnBinary, "[openvpn binary check] ")
-	if err := process.Start([]string{"--version"}); err != nil {
-		return err
-	}
-	cmdResult := process.Wait()
-
+	command := exec.Command(openvpnBinary, "--version")
+	outputBuffer, cmdResult := command.Output()
 	exitCode, err := extractExitCodeFromCmdResult(cmdResult)
 	if err != nil {
 		return err
 	}
-
 	//openvpn returns exit code 1 in case of --version parameter, if anything else is returned - treat as error
 	if exitCode != 1 {
-		return errors.New("unexpected openvpn code: " + strconv.Itoa(exitCode))
+		log.Error("[Openvpn check] ", "Check failed. Output of executed command: ", string(outputBuffer))
```

Comment:
"Openvpn check" we could move this to separate prefix.

Reply:
move it to constant

Reply:
Moved